PowerEditor/src/langs.model.xml

```diff
@@ -239,9 +239,9 @@
-            <Keywords name="type1">ARCHIVE CUR END FILE_ATTRIBUTE_ARCHIVE FILE_ATTRIBUTE_HIDDEN FILE_ATTRIBUTE_NORMAL FILE_ATTRIBUTE_OFFLINE FILE_ATTRIBUTE_READONLY FILE_ATTRIBUTE_SYSTEM FILE_ATTRIBUTE_TEMPORARY HIDDEN HKCC HKCR HKCU HKDD HKEY_CLASSES_ROOT HKEY_CURRENT_CONFIG HKEY_CURRENT_USER HKEY_DYN_DATA HKEY_LOCAL_MACHINE HKEY_PERFORMANCE_DATA HKEY_USERS HKLM HKPD HKU IDABORT IDCANCEL IDIGNORE IDNO IDOK IDRETRY IDYES MB_ABORTRETRYIGNORE MB_DEFBUTTON1 MB_DEFBUTTON2 MB_DEFBUTTON3 MB_DEFBUTTON4 MB_ICONEXCLAMATION MB_ICONINFORMATION MB_ICONQUESTION MB_ICONSTOP MB_OK MB_OKCANCEL MB_RETRYCANCEL MB_RIGHT MB_SETFOREGROUND MB_TOPMOST MB_USERICON MB_YESNO MB_YESNOCANCEL NORMAL OFFLINE READONLY SET SHCTX SW_HIDE SW_SHOWMAXIMIZED SW_SHOWMINIMIZED SW_SHOWNORMAL SYSTEM TEMPORARY all auto both bottom bzip2 checkbox colored current false force hide ifdiff ifnewer lastused leave left listonly lzma nevershow none normal off on pop push radiobuttons right show silent silentlog smooth textonly top true try zlib</Keywords>
+            <Keywords name="type1">3DFACE ACP ARCHIVE BOM BRANDING BTNTEXT COMPONENTSONLYONCUSTOM CUR CUSTOMSTRING ENABLECANCEL END EXERESOURCE FILESONLY FINAL GETDLGITEM GRAYTEXT HIDDEN HIGHLIGHT HIGHLIGHTTEXT HKCC HKCR HKCR32 HKCR64 HKCRANY HKCU HKCU32 HKCU64 HKCUANY HKDD HKEY_CLASSES_ROOT HKEY_CURRENT_CONFIG HKEY_CURRENT_USER HKEY_DYN_DATA HKEY_LOCAL_MACHINE HKEY_PERFORMANCE_DATA HKEY_USERS HKLM HKLM32 HKLM64 HKLMANY HKPD HKU HOTLIGHT IDABORT IDCANCEL IDD_DIR IDD_INST IDD_INSTFILES IDD_LICENSE IDD_LICENSE_FSCB IDD_LICENSE_FSRB IDD_SELCOM IDD_UNINST IDD_VERIFY IDIGNORE IDNO IDOK IDRETRY IDYES IMGID ITALIC LANG MB_ABORTRETRYIGNORE MB_DEFBUTTON1 MB_DEFBUTTON2 MB_DEFBUTTON3 MB_DEFBUTTON4 MB_ICONEXCLAMATION MB_ICONINFORMATION MB_ICONQUESTION MB_ICONSTOP MB_OK MB_OKCANCEL MB_RETRYCANCEL MB_RIGHT MB_RTLREADING MB_SETFOREGROUND MB_TOPMOST MB_USERICON MB_YESNO MB_YESNOCANCEL NOCUSTOM NOERRORS NORMAL NoBOM NoWorkingDir OEM OFFLINE OVERWRITE READONLY REGEDIT5 REPLACE RESET RESIZETOFIT RESIZETOFITHEIGHT RESIZETOFITWIDTH RawNL SET SHCTX SHCTX32 SHCTX64 SHCTXANY SHELL_CONTEXT SHORT SILENT SOLID STRIKE STRINGID SW_HIDE SW_SHOW SW_SHOWDEFAULT SW_SHOWMAXIMIZED SW_SHOWMINIMIZED SW_SHOWMINNOACTIVE SW_SHOWNA SW_SHOWNORMAL SYSTEM TEMPORARY TIMEOUT UNDERLINE UTF16BE UTF16BOM UTF16LE UTF8 UTF8SIG WINDOW WINDOWTEXT admin all alwaysoff amd64 ansi arm64 auto both bottom bzip2 code colored components current custom default directory disable enable error explorer false fileexists force gray grey hide highest ifdiff ifempty ifnewer instfiles lastused left license listonly lzma manual nevershow noerrors none nonfatal normal notext notset off on oname packed permonitor plugin pop push right show silent silentlog smooth system textonly top true try un.components un.custom un.directory un.instfiles un.license un.uninstConfirm unicode uninstConfirm user warning windows x64 x86 zlib</Keywords>

```

Comment:
Just check **3DFACE** and it doesn't seem the keyword of NSIS.
Could you provide any link or ref of NSIS keywords?